res/openvpn-gui-res-it.rc

```diff
@@ -371,34 +402,40 @@ BEGIN
-    IDS_ERR_AUTH_PASSWORD2STDIN "Errore nel trasmettere la password a stdin."
+    IDS_ERR_PASSPHRASE2STDIN "Errore nella trasmissione della frase di sicurezza a stdin."
+    IDS_ERR_AUTH_USERNAME2STDIN "Errore nella trasmissione del nome utente a stdin."
+    IDS_ERR_AUTH_PASSWORD2STDIN "Errore nella trasmissione della password a stdin."
     IDS_ERR_CR2STDIN "Errore nel trasmettere CR a stdin."
     IDS_ERR_INVALID_CHARS_IN_PSW "La tua nuova password contiene caratteri non validi. " \
                                  "Per favore scegline un'altra."
 
     /* settings */
-    IDS_SETTINGS_CAPTION "OpenVPN - Configurazione"
+    IDS_SETTINGS_CAPTION "OpenVPN - Impostazioni"
 
     /* proxy */
-    IDS_ERR_HTTP_PROXY_ADDRESS "Devi specificare un indirizzo per il HTTP proxy."
-    IDS_ERR_HTTP_PROXY_PORT "Devi specificare una porta per il HTTP proxy."
-    IDS_ERR_HTTP_PROXY_PORT_RANGE "Devi specificare una porta tra 1-65535 per il HTTP proxy port between 1-65535"
-    IDS_ERR_SOCKS_PROXY_ADDRESS "Devi specificare un indirizzo per il SOCKS proxy."
-    IDS_ERR_SOCKS_PROXY_PORT "Devi specificare una porta per il SOCKS proxy."
-    IDS_ERR_SOCKS_PROXY_PORT_RANGE "Devi specificare una porta tra 1-65535 per il SOCKS proxy"
+    IDS_ERR_HTTP_PROXY_ADDRESS "Devi specificare un indirizzo per il proxy HTTP."
+    IDS_ERR_HTTP_PROXY_PORT "Devi specificare una porta per il proxy HTTP."
+    IDS_ERR_HTTP_PROXY_PORT_RANGE "Devi specificare una porta tra 1-65535 per il proxy HTTP."
+    IDS_ERR_SOCKS_PROXY_ADDRESS "Devi specificare un indirizzo per il proxy SOCKS."
+    IDS_ERR_SOCKS_PROXY_PORT "Devi specificare una porta per il proxy SOCKS."
+    IDS_ERR_SOCKS_PROXY_PORT_RANGE "Devi specificare una porta tra 1-65535 per il proxy SOCKS."
     IDS_ERR_CREATE_REG_HKCU_KEY "Errore nella creazione della chiave ""HKEY_CURRENT_USER\\%s""."
     IDS_ERR_GET_TEMP_PATH "Errore nel determinare TempPath con GetTempPath(). Usa ""C:\\"" invece."
 
     /* service */
-    IDS_ERR_OPEN_VPN_SERVICE "Errore nell'apertura di ""OpenVPNService"""
-    IDS_ERR_START_SERVICE "Errore nel avvio di ""OpenVPNService"""
+    IDS_ERR_OPEN_VPN_SERVICE "Errore nell'apertura del servizio OpenVPN"
+    IDS_ERR_START_SERVICE "Errore nel avvio del servizio OpenVPN"
     IDS_ERR_QUERY_SERVICE "Errore nell'interrogazione dello status del servizio."
-    IDS_ERR_SERVICE_START_FAILED "Fallito l'avvio di OpenVPN Service."
+    IDS_ERR_SERVICE_START_FAILED "Errore nell'avvio del servizio OpenVPN"
     IDS_ERR_OPEN_SCMGR "OpenSCManager fallito (%d)"
-    IDS_ERR_STOP_SERVICE "Fallito l'arresto di OpenVPN Service"
-    IDS_NFO_RESTARTED "OpenVPN Service riavviato."
+    IDS_ERR_STOP_SERVICE "Errore nell'arresto del servizio OpenVPN"
+    IDS_NFO_RESTARTED "Servizio OpenVPN riavviato."
+    IDS_ERR_ACCESS_SERVICE_PIPE "Accesso alla pipe del servizio fallito."
+    IDS_ERR_WRITE_SERVICE_PIPE "Scrittura alla pipe del servizio fallita."
```

Comment:
What do you think about "sulla pipe" instead of "alla pipe" ? But I am not even sure here. Normally I'd say "scrivi sul socket", so I thought about the same for a pipe.

Reply:
I'll change the row 434 occurrence as your version does sound better for that operation.